packages/flutter/test/painting/_network_image_test_web.dart

```diff
@@ -92,4 +92,27 @@ void runTests() {
     expect(tester.takeException().toString(),
         'HTTP request failed, statusCode: 200, https://www.example.com/images/frame3.png');
   });
+
+  testWidgets('loads an image using withCredentials true',
+      (WidgetTester tester) async {
+    if (!isCanvasKit) {
+      // The HTML renderer uses an `<img>` tag, it does not fetch the image via XMLHttpRequest.
+      return;
```

Comment:
Would it be better to use https://pub.dev/documentation/test/latest/index.html#skipping-tests ??

Reply:
Or maybe not. It adds noise to the output...